cms/envs/common.py

```diff
@@ -1561,6 +1566,14 @@
 # (0.0 = 0%, 1.0 = 100%)
 COMPLETION_VIDEO_COMPLETE_PERCENTAGE = 0.95
 
+############## Settings for colaraz_features app ##############
+# [COLARAZ_CUSTOM]
+
+# These variables are used in ColarazAuthenticationMiddleware
+COLARAZ_ENABLE_AUTH_MIDDLEWARE = False
```

Comment:
```suggestion
ENABLE_COLARAZ_AUTH_MIDDLEWARE = False
```
Just to be consistent with edx settings.